core/domain/role_services.py

```diff
@@ -49,18 +51,21 @@
 ACTION_CREATE_NEW_TOPIC = 'CREATE_NEW_TOPIC'
 ACTION_MANAGE_QUESTION_SKILL_STATUS = 'MANAGE_QUESTION_SKILL_STATUS'
 ACTION_DELETE_ANY_ACTIVITY = 'DELETE_ANY_ACTIVITY'
+ACTION_DELETE_ANY_BLOG_POST = 'DELETE_ANY_BLOG_POST'
 ACTION_DELETE_ANY_PUBLIC_ACTIVITY = 'DELETE_ANY_PUBLIC_ACTIVITY'
 ACTION_DELETE_ANY_QUESTION = 'DELETE_ANY_QUESTION'
 ACTION_DELETE_ANY_SKILL = 'DELETE_ANY_SKILL'
+ACTION_DELETE_OWNED_BLOG_POST = 'DELETE_BLOG_POST'
 ACTION_DELETE_OWNED_PRIVATE_ACTIVITY = 'DELETE_OWNED_PRIVATE_ACTIVITY'
 ACTION_DELETE_TOPIC = 'DELETE_TOPIC'
 ACTION_EDIT_ANY_ACTIVITY = 'EDIT_ANY_ACTIVITY'
+ACTION_EDIT_ANY_BLOG_POST = 'EDIT_ANY_BLOG_POST'
 ACTION_EDIT_ANY_PUBLIC_ACTIVITY = 'EDIT_ANY_PUBLIC_ACTIVITY'
 ACTION_EDIT_ANY_QUESTION = 'EDIT_ANY_QUESTION'
 ACTION_EDIT_ANY_SKILL = 'EDIT_ANY_SKILL'
 ACTION_EDIT_ANY_SUBTOPIC_PAGE = 'EDIT_ANY_SUBTOPIC_PAGE'
 ACTION_EDIT_ANY_TOPIC = 'EDIT_ANY_TOPIC'
-ACTION_EDIT_ANY_BLOG_POST = 'EDIT_ANY_BLOG_POST'
+ACTION_EDIT_OWNED_BLOG_POST = 'EDIT_BLOG_POST'
```

Comment:
Similarly, where is this used?

Some other newly-added actions below seem superfluous, or at least I can't find where they're used. Could you please double-check and remove the ones that aren't being used (or explain why they're here)?

Reply:
Done.Removed